synapse/lib/ast.py

```diff
@@ -1723,8 +1736,11 @@ class CallArgs(RunValue):
     async def runtval(self, runt):
         return [await k.runtval(runt) for k in self.kids]
 
-class CallKwarg(CallArgs): pass
-class CallKwargs(CallArgs): pass
+class CallKwarg(CallArgs):
```

Comment:
why are these changes in this delta?

Reply:
these are the last changes necessary to make this file PEP8.